src/cinema/businesslayer/CinemaService.java

```diff
@@ -0,0 +1,55 @@
+package cinema.businesslayer;
+
+import cinema.domainmodel.Seat;
+import cinema.domainmodel.Seats;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class CinemaService implements CommandLineRunner {
+
+    private final int numOfRows = 9;
+    private final int numOfColumns = 9;
+    private Seats seats;
+
+    private void generateSeats(){
+        List<Seat> seatsArr = new ArrayList<>();
+        for (int i = 1; i <= numOfRows; i++){
```

Comment:
(minor) Usually we start indexing from 0

Reply:
Numbers of rows and columns are used for identifying the exact seat/ticket by customers and the numeration for them usually starts with 1 in cinemas